engines/sci/detection_tables.h

```diff
@@ -1133,7 +1133,7 @@ static const struct ADGameDescription SciGameDescriptions[] = {
 		Common::EN_ANY, Common::kPlatformMacintosh, ADGF_MACRESFORK, GUIO4(GUIO_NOSPEECH, GAMEOPTION_PREFER_DIGITAL_SFX, GAMEOPTION_ORIGINAL_SAVELOAD, GAMEOPTION_FB01_MIDI)	},
 
 #ifdef ENABLE_SCI32
-#define GUIO_HOYLE5_DEMO GUIO3(GUIO_NOSPEECH, \
+#define GUIO_HOYLE5 GUIO3(GUIO_NOSPEECH, \
                                GUIO_NOASPECT, \
                                GAMEOPTION_ORIGINAL_SAVELOAD)
```

Comment:
Some games that do not use savegames replace Game::save and Game::restore with stubs. In that case replacing save/load screens may cause script corruption. Might be worth checking this first. m_kiewitz knows much more about it than me, though.